node/src/chain_spec/cs_staking.rs

```diff
@@ -211,6 +211,118 @@ pub fn development_config() -> ChainSpec {
     )
 }
 
+fn tnet_templ_staking_config_genesis() -> GenesisConfig {
```

Comment:
For the "template" (usually called "local" for the other chain specs) we can actually stick to the Alice, Bob, etc... keys. Then for the production spec (the JSON) have actual production keys there. This ensures that third parties can still run the local spec.

Reply:
@Eliott, already we have "staking-local" with "Alice & Bob",
Introduced "staking-tnet" & "staking-tnet-templ" extra to bringup
staking testnet based on private keys. Is my understanding correct ?

| Chain-Spec | Discp |
| --- | --- |
| "staking-dev" | One node development setup, with "Alice" as authority |
| "staking-local" | Two node local setup with "Alice" & "Bob" as authority |
| "staking-tnet" | Multi-node test network with user private keys as authority, & load's chainspec from "node/res/tnet-staking.json" |
| "staking-tnet-templ" | Same as "staking-tnet", but used to generate the "node/res/tnet-staking.json" spec file. |

Reply:
Replaced "staking-tnet" to "staking" & removed "staking-tnet-templ"

| Chain-Spec | Status | Discp |
| --- | --- | --- |
| "staking-dev" | Existing One | One node development setup, with "Alice" as authority |
| "staking-local" |  Existing One | Two node local setup with "Alice" & "Bob" as authority |
| "staking" | New One | Multi-node test network with user private keys as authority, & load's chainspec from "node/res/tnet-staking.json" |